src/server.js

```diff
@@ -0,0 +1,63 @@
+const ipc = require('electron').ipcRenderer;
+const express = require('express');
+const log = require('electron-log');
+const {Validator} = require('express-json-validator-middleware');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const path = require('path');
+const app = express();
+
+const {getLogTransportConsole, getLogTransportFile, getLogResolvePath} = require('./lib/logFormat');
+const ConstructURL = require('./utils/ConstructURL');
+const { validate } = new Validator();
+
+
+window.addEventListener('DOMContentLoaded', () => {
+    // Method to format the writings logs
+    getLogTransportConsole();
+    getLogTransportFile();
+    getLogResolvePath();
+
+
+    console.log('Server started');
+
+    app
+    .use(cors())
+    .use(bodyParser.json())
+    .use(express.static(path.join(__dirname, '../static')))
+    .use(express.static(path.join(__dirname, '../static/help/assets/style')))
```

Comment:
Is it mandatory to declare all sub folders?

Reply:
yes because if i don't do that the css file is not recognized by the viewer.